examples/next/pages/_app.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import {MDXProvider} from '@mdx-js/react'
 
-const mdComponents = {
+const components = {
   h1: props => <h1 style={{color: 'tomato'}} {...props} />
 }
 
-export default ({Component, pageProps}) => (
-  <MDXProvider components={mdComponents}>
-    <Component {...pageProps} />
+const Page = ({Component}) => (
+  <MDXProvider components={components}>
+    <Component />
   </MDXProvider>
 )
+
+export default Page
```

Comment:
exports must have an inferrable name according to next

Reply:
1. isn't that a requirement for fast refresh, not for Next as a whole. (https://nextjs.org/docs/advanced-features/codemods#nextjs-9)
2. arrow functions don't have names, to five this a name wouldn't it need to be `function Page ({Component}) {`?

Reply:
For 2, I think v8 does infer the .name of arrow functions in a variable now? Or they're walking the AST.
For 1, probably. It was a warning and they suggested this, so I applied it

Reply:
huh, that seems new, IIRC originally arrow functions never had a name. :man_shrugging: 
Ah well it works now, good to know.